docs/proposals/Drawer/README.md

```diff
@@ -0,0 +1,70 @@
+# Drawer
+
+The `<Drawer>` component will provide a dismissible side panel.
+
+## Design Patterns
+
+`<Drawer>` It will allow the user to view related content to their main task.
+Unlike a `<Modal>`, it will take full advantage of the available viewport
+height. This will make it ideal for long lists and other scrolling content.
```

Comment:
> Unlike a `<Modal>`, it will take full advantage of the available viewport height. This will make it ideal for long lists and other scrolling content.
This might be nitpicking, but could a `Modal` with a max-height and better handling of vertical overflow solve the same problem? Right now the behaviour of dialogs/modals in Jobber is that the entire view scrolls with the modal if "contentHeight" > "viewportHeight" which feels to me like the key difference being described here.